lib/build.rb

```diff
@@ -78,13 +77,13 @@ def query_branches
         days = (DateTime.now - DateTime.parse(branch_details.commit.commit.author.date.to_s)).round
         if days <= @max_age
           login = 'Unknown'
-          if !branch_details.commit.author.nil?
-            login = branch_details.commit.author.login
-          else
+          if branch_details.commit.author.nil?
```

Comment:
Better if condition order in several spots in these changes.  I never understood why Rubocop wanted it the other way in the past, but I went with it.